ehr/resources/queries/ehr/kinshipSummary.sql

```diff
@@ -3,19 +3,6 @@
  *
  * Licensed under the Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0
  */
-SELECT
-  k.Id,
-  k.Id2,
-  k.coefficient
 
-FROM ehr.kinship k
-
-UNION ALL
-
-SELECT
-  k.Id2 as Id,
-  k.Id as Id2,
-  k.coefficient
-
-FROM ehr.kinship k
-WHERE k.Id != k.Id2
\ No newline at end of file
+SELECT * FROM ehr.kinship
+WHERE Id != Id2
```

Comment:
isnt this defeating the entire point of this script???

Reply:
I'm not sure how it's defeating the point of the script. It's not really necessary other than there are reports in some of the PRCs that are using this query so I don't want to pull the rug out from under those. Before this change it was causing duplicate rows with the new calculations.

I'll file tickets to get those usages refactored to just use ehr.kinship but that will likely not be in this PR.

Reply:
Yes that's all true. Kinship coefficient calculates for Ids in both directions. That's just the default way kinship2 is calculating the pedigree then the kinship, so don't need to do the union reversing the rows.  And yes the animal history kinship report will need to be updated to just use ehr.kinship for any centers using ehr.kinshipSummary.